aztec/src/main/kotlin/org/wordpress/aztec/AztecText.kt

```diff
@@ -673,7 +673,7 @@ class AztecText : AppCompatAutoCompleteTextView, TextWatcher, UnknownHtmlSpan.On
         }
     }
 
-    fun setToolbar(toolbar: AztecToolbar) {
+    fun setToolbar(toolbar: AztecToolbar?) {
```

Comment:
This does not need to be nullable, since this call will always be called with a non-null value.